extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js

```diff
@@ -90,7 +94,7 @@ define('wikia.articleVideo.featuredVideo.events', function () {
 		playerInstance.on('play', function (data) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "data". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)